pkg/cri/server/helpers.go

```diff
@@ -510,3 +510,11 @@ func copyResourcesToStatus(spec *runtimespec.Spec, status containerstore.Status)
 	}
 	return status
 }
+
+func (c *criService) sendContainerEvent(event runtime.ContainerEventResponse) {
+	select {
+	case c.containerEventsChan <- event:
+	default:
+		logrus.Warnf("containerEventsChan is full, discarding event %+v", event)
```

Comment:
// TODO: write events to a cache, storage, or increase the size of the channel 

for now let's reduce the spam some by making this debug or at least limiting it to the first full until a successful send..